src/main/scala/com/qubole/sparklens/common/AggregateMetrics.scala

```diff
@@ -67,6 +71,7 @@ object AggregateValue {
     value.max = (json \ "max").extract[Long]
     value.mean = (json \ "mean").extract[Double]
     value.variance = (json \ "variance").extract[Double]
+    value.m2 = (json \ "m2").extract[Double]
```

Comment:
Please make this optional. It will fail with existing sparklens.json files. Rest is good. Thanks
```
//making it optional for backward compatibility with sparklens.json files
    value.m2 = (json \ "m2").extractOrElse[Double](0.0)
```

Reply:
Indeed! I added your suggestion, closing this discussion.